src/Application/DefaultPreset.php

```diff
@@ -49,11 +50,16 @@ public static function get(): array
                 UnusedVariableSniff::class => [
                     'ignoreUnusedValuesWhenOnlyKeysAreUsedInForeach' => true,
                 ],
+                PropertyTypeHintSniff::class => [
+                    'enableNativeTypeHint' => optional($composer)->lowestPhpVersionMatches('7.4') === true ? false : null,
```

Comment:
what is this function `optional` ?

Reply:
It's again something from illuminate/support, it basically just says makes the method you are chaining on return null if the object you are wrapping in optional is null 👍 
Can easily be replaced with something not from illuminate/support